### DISCUSSION TITLE
Store data from a sheet into variable

### DISCUSSION THREAD

**Discussion by JoeLumenElectronics** [mobizt/ESP-Google-Sheet-Client#31] [Q&A]
Hello, 
I am wanting to create a stock display that will display the stock from my google sheet on a screen, to do so i need to be able to store the data from a range of cells into various variables, however when i use the GSheet.values.get command i receive more than just the value within cell and therefore unable to store it within a variable. is it possible to do so? if so, how?

**Comment by mobizt** [comment]
You should read the Google sheet API doc and try. 
If you understand spreadsheet or excel, the range value can be a cell to multi-row and multi-column.
